index.rst

```diff
@@ -13,13 +13,93 @@
+That is to say, if daytime engineering work within AuxTel prevents us from starting calibrations and venting before 16:00, these tasks can be skipped at the discretion of the daytime observing specialist.
+
+Nighttime Activities
+====================
+
+Nighttime Operations
+--------------------
+The AuxTel should be entirely scheduler-driven at night while in survey mode. 
+A single scheduler configuration should be provided each night to prevent detailed night plans requiring active observing support.
+Once the daytime activities are complete, the telescope can be prepare for on-sky observations and the scheduler configured at any time before twilight. 
+The AuxTel should continue to be run during the night until the observing team on the summit deems it necessary to close.
+
+Fault resolution
+----------------
+Faults which interrupt observations in this mode shall be recovered when the observing specialists on the summit have time and are not occupied by responsibilities related to Simonyi Survey Telescope operations. 
+When faults occur which result only in script failures that do not result in hardware risk or CSC components going into fault state, operations should be recovered by simply playing the script queue. 
+The scheduler and standard scripts used during observations are required to be robust against such failure, and should not leave the system in an inoperable state in the case of script failure alone.
+Faults that result in CSC components going into fault state and which require intervention from the observing specialists to recover will be recovered when possible. 
+CSC faults may result in the system remaining inoperable for extended periods of time if the observing specialists are unavailable to recover. 
+In this case, the CSCs in fault should be transitioned to Standby while they await availability to resolve to prevent warning messages being broadcast. 
+Remote support can be utilized to recover the system with explicit approval from the observing specialists on shift. 
+
+Roles and Responsibilities
+==========================
+
+Daytime Observing Specialist Shift
+----------------------------------
+The Daytime Shift is expected to attend the 09:30 morning meeting and report on any known failures that require daytime resources.
+They are also expected to follow any on-going work during the day. 
+They are responsible for deciding if/when it is appropriate to start daytime calibrations and venting. 
+If Simonyi Survey telescope responsibilities prevent the Daytime Shift from completing calibrations and venting, they may be skipped. 
+
+Flex/Nighttime Observing Specialist Shift
+-----------------------------------------
+The Nighttime Shift is responsible for passively monitoring AuxTel status throughout the night. 
+This does not include monitoring data quality, but only operational status. 
```

Comment:
I would still encourage that an eyeball pass over the last x-number of image output statistics on Rubintv IF POSSIBLE every once in a while. Crappy images are better flagged at night and make it much easier for processing later. This ask may amount to 5 minutes every hour, and images can be flagged after the fact.